packages/client/hmi-client/src/workflow/tera-workflow.vue

```diff
@@ -263,6 +263,9 @@ import {
 
 import { TeraCodeAssetNode, CodeAssetOperation, TeraCodeAssetWrapper } from './ops/code-asset/mod';
 
+// New import style for operators
+import * as ModelConfigOp from './ops/model-config/mod';
```

Comment:
I wonder if we should move the `/ops` outside of `/worflow`.
Or update [vite.config.ts alias](https://github.com/DARPA-ASKEM/terarium/blob/main/packages/client/hmi-client/vite.config.ts#L27-L33) with a `@ops` to avoid using the relative path.

Reply:
So what is your point of view on this question? I see it mark as resolved but nothing in the code